tools/convert_partition.py

```diff
@@ -218,7 +218,7 @@ def read_feats(file_name):
                  'etypes': etypes_map}
 
 for part_id in range(num_parts):
-    part_dir = output_dir + '/part' + str(part_id)
+    part_dir = 'part' + str(part_id)
```

Comment:
`tools/convert_partition.py` was added yesterday, is there any doc / example of how to use it?